### DISCUSSION TITLE
GIFT CARD

### DISCUSSION THREAD

**Discussion by jawed1412** [bagisto/bagisto#7221] [Ideas]
Need giftcard

**Comment by Sanjaybhattwebkul** [comment]
Hi, there 
Are you asking about the gift card feature in bagisto?

**Reply by jawed1412** [reply]
yes it will be very helpful

**Comment by Sanjaybhattwebkul** [comment]
Hi there,
For now this feature is not available in our latest release, but yes we will introduce this feature in our future release.

**Reply by jawed1412** [reply]
Thankyou for the consideration
